app/forms/idp/create.tsx

```diff
@@ -120,7 +122,36 @@ export function CreateIdpSideModalForm() {
       submitError={createIdp.error}
       submitLabel="Create provider"
     >
-      <NameField name="name" control={form.control} />
+      <Message
+        content={
+          <>
+            Read the{' '}
+            <a
+              href={links.identityProvidersDocs}
+              className="underline"
+              target="_blank"
+              rel="noreferrer"
+            >
+              Rack Configuration
+            </a>{' '}
+            guide to learn more about setting up an identity provider.
+          </>
+        }
+      />
+      <NameField
+        name="name"
+        control={form.control}
+        description={
+          <>
+            A short name for the provider in our system. Users will see it in the path to
+            the login page:{' '}
+            <code>
+              /login/{silo}/saml/{name.trim() || 'idp-name'}
+            </code>
+            .
```

Comment:
If we're ending the copy with `<code>`, I think it looks better (and clearer) without the ultimate period. Thoughts about leaving that off?

Reply:
I have it in there because it’s two sentences. Otherwise I would leave it out.

Reply:
Hm, yeah I think you’re right. I’ll take it out.